module/users/validator/v1.js

```diff
@@ -12,6 +12,10 @@ module.exports = (req) => {
         targetedEntity : function () {
             req.checkParams("_id").exists().withMessage("required solution id");
             req.checkBody("role").exists().withMessage("required user role");
+        },
+        getAllStats:function(){
+            req.checkQuery('type').optional().isIn(['observation', 'project', 'survey', 'program']) // If present, it must be one of these values
```

Comment:
this not required for big number @borkarsaish65

Reply:
this is optional as you can see options() is used ,same api is used to list big numbers and detailed stats